test/integration/benchmark/jsonify/main.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	benchparse "golang.org/x/tools/benchmark/parse"
+	"k8s.io/kubernetes/test/e2e/perftype"
+)
+
+func main() {
+	err := run()
+	if err != nil {
+		panic(err)
+	}
+}
+
+func run() error {
+	if len(os.Args) < 2 {
+		return fmt.Errorf("output filename is a required argument")
+	}
+	benchmarkSet, err := benchparse.ParseSet(os.Stdin)
+	if err != nil {
+		return err
+	}
+	data := perftype.PerfData{Version: "v1"}
+	for _, benchMarks := range benchmarkSet {
+		for _, benchMark := range benchMarks {
+			data.DataItems = appendIfMeasured(data.DataItems, benchMark, benchparse.NsPerOp, "time", "μs", benchMark.NsPerOp/1000.0)
```

Comment:
μs seems to be printed weirdly as a unicode character - https://storage.googleapis.com/kubernetes-jenkins/logs/pull/59800/ci-benchmark-scheduler-master/20180214-131137--3d30881ee7faa2da-17/artifacts/BenchmarkResults_benchmark_2018-02-14T13:12:45Z.txt

Maybe use sth else?

Reply:
That's because storage.googleapis.com web frontend declares content-type as just `text/plain`, without specifying the encoding. If you download it and open it in an UTF-8-aware editor it will look fine.

Reply:
SG

Reply:
But I don't want to do such things to see the results in clear format.

Can you change that to "us" - I think it's commonly used in many places.